### PR TITLE
The coffee machines uses telnet not SSH

### PR DESCRIPTION
**Pull request by mhsjlw**
According to `coffee.rb` it uses [sys brew](https://github.com/NARKOZ/hacker-scripts/blob/master/fucking_coffee.rb#L18) and [sys pour](https://github.com/NARKOZ/hacker-scripts/blob/master/fucking_coffee.rb#L20) and uses [telnet](https://github.com/NARKOZ/hacker-scripts/blob/master/fucking_coffee.rb#L16)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ story](https://www.jitbit.com/alexblog/249-now-thats-what-i-call-a-hacker/)_:
...
 
-> xxx: (and the oscar goes to) [`fucking-coffee.sh`](https://github.com/NARKOZ/hacker-scripts/blob/master/fucking-coffee.sh) - this one waits exactly 17 seconds (!), then opens an SSH session to our coffee-machine (we had no frikin idea the coffee machine is on the network, runs linux and has SSHD up and running) and sends some weird gibberish to it. Looks binary. Turns out this thing starts brewing a mid-sized half-caf latte and waits another 24 (!) seconds before pouring it into a cup. The timing is exactly how long it takes to walk to the machine from the dudes desk.
+> xxx: (and the oscar goes to) [`fucking-coffee.sh`](https://github.com/NARKOZ/hacker-scripts/blob/master/fucking-coffee.sh) - this one waits exactly 17 seconds (!), then opens an telnet session to our coffee-machine (we had no frikin idea the coffee machine is on the network, runs linux and has a TCP socket up and running) and sends something like `sys brew`. Turns out this thing starts brewing a mid-sized half-caf latte and waits another 24 (!) seconds before pouring it into a cup. The timing is exactly how long it takes to walk to the machine from the dudes desk.
```
Comment (niroyb) [inline README.md]: Replace "an telnet" by "a telnet"

Comment (xTear) [inline README.md]: I want to know which coffee machine is used..

**Comment by azeemhassni** [comment]
+1 I want to know which coffee machine is used..

**Comment by mhsjlw** [comment]
Alright, @niroyb I fixed up the grammar mistake.

**Comment by NARKOZ** [comment]
Please squash the commits into one.

**Comment by marcelosevero** [comment]
what is the coffee machine brand?

**Comment by mhsjlw** [comment]
Ok, I clearly can't squash commits.

**Comment by aleksandar-todorovic** [comment]
@mhsjlw Maybe reimplement the changes and commit them all at once as a new PR?

**Comment by NARKOZ** [comment]
ac6edc5

**Pull request closed without merging**